plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialMode.java

```diff
@@ -132,6 +203,21 @@ else if (isBubbleColumn()) {
             else if (isPistonHead()) {
                 getPistonHead().setShort(CoreUtilities.equalsIgnoreCase(mechanism.getValue().asString(), "short"));
             }
+            else if (isStructureBlock() && mechanism.requireEnum(false, StructureBlock.Mode.values())) {
+                getStructureBlock().setMode(StructureBlock.Mode.valueOf(mechanism.getValue().asString().toUpperCase()));
+            }
+            else if (isSculkSensor() && mechanism.requireEnum(false, SculkSensor.Phase.values())) {
+                ((SculkSensor) material.getModernData()).setPhase(SculkSensor.Phase.valueOf(mechanism.getValue().asString().toUpperCase())); // TODO: 1.17

```

Comment:
requires the `MultiVersionHelper1_17` thing as shown for example with dripstone in material type